.env

```diff
@@ -1,3 +1,3 @@
 REACT_APP_CHAIN_ID="1"
 REACT_APP_GOOGLE_ANALYTICS_MEASUREMENT_ID=""
-REACT_APP_NETWORK_URL=""
+REACT_APP_NETWORK_URL="https://eth-mainnet.alchemyapi.io/v2/qNhZ_-GaDehhI-HSlG_dff9nxS6nuXGg"
```

Comment:
this should be set in an `.env.local` file so it isn't committed to git.  in general, you never want to commit secrets or keys to repositories.

Reply:
fixed